shopsafe-frontend/src/app/components/landing/landing.component.ts

```diff
@@ -11,13 +11,16 @@ import { Router } from "@angular/router";
 export class LandingComponent implements OnInit {
 
   location: string = '';
+  loadingUserLatLng: boolean = false;
```

Comment:
So I created a similar boolean, I'm guessing there is a conflict there.

Reply:
i use the boolean i created as a way to tell me if im currently trying to get user location, this way i can show the user that we're getting their location and not have that weird buffer time and regular screen where they can't tell if the button work. But it shouldn't conflict with your changes